cores/esp8266/umm_malloc/umm_info.c

```diff
@@ -36,6 +37,7 @@ void *umm_info( void *ptr, int force ) {
    */
   memset( &ummHeapInfo, 0, sizeof( ummHeapInfo ) );
 
+  DBGLOG_FORCE( force, "\n" );
```

Comment:
1. For `umm_info(NULL, true)` printing a heap analysis - does it need an ISR safe print method?
All printing is done from within a critical section with `rsil(15)`.
2. And/Or should we have the print option disabled for non-debug builds?
ie. `Debug port: "Disabled"`

Reply:
@devyte To implement `system_show_malloc()` in `heap.cpp` a call to `umm_info(NULL, 1);` was/is made. Is it an okay thing to have this only work for a debug build or if the build is done with `-DUMM_INFO_PRINT` option specified.